manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension
+License: MIT License
+Copyright: Copyright © 1999-2020
+ShortDescription: Link Shellextension

```

```suggestion
ShortDescription: Link Shell Extension provides for the creation of Hardlinks, Junctions, Volume Mountpoints, and Symbolic Links, a folder cloning process that utilizes Hardlinks / Symbolic Links and a copy process taking care of Junctions, Symbolic Links, and Hardlinks.
```
ShortDescription should be meaningful; this is what I got from the website and I've tried to shorten it to meet the maximum 256 characters requirement. Please commit if you agree with this description.